www/perl-lib/SMT/Rest/SCCAPIv4.pm

```diff
@@ -355,6 +355,14 @@ sub delete_single_product
 {
     my $self = shift || return (undef, undef);
 
+    my $regsharing = SMT::Utils::hasRegSharing();
```

Comment:
Did mc identify this, or are you now aware of any other business logic cases disallowing deregistration that I'm not aware of as a non-specialist?